Vungle/VungleRouter.m

```diff
@@ -46,8 +47,11 @@ @interface VungleRouter ()
 @property (nonatomic) SDKInitializeState sdkInitializeState;
 
 @property (nonatomic) NSMutableDictionary *waitingListDict;
+@property (nonatomic) NSMutableDictionary *hbWaitingListDict;
```

Comment:
@stanley-vungle We should discuss the addition of this dictionary and the map tables below - if there is an understanding that the key can be either placement ID or event ID, then we should use the original objects. There is no reason we should have to split these into separate objects if we are certain there will be no duplication of placement IDs or event IDs.

Reply:
From the meeting, it was agreed upon that the hb and non-hb dictionaries can be merged.

The keys of the dictionaries will take the format of `placement|event`.
In the case of non-hb delegates, the event id will have nothing in the place of the event id like `placement|`

The reason for the key format is IOS-3483 where delegates will need to be removed by placement id if something changes mid-session to invalidate an ad.